diff --git a/node_modules/@nestjs/common/services/logger.service.d.ts b/node_modules/@nestjs/common/services/logger.service.d.ts
index 8e52c68..89ab4f1 100644
--- a/node_modules/@nestjs/common/services/logger.service.d.ts
+++ b/node_modules/@nestjs/common/services/logger.service.d.ts
@@ -21,11 +21,11 @@ export interface LoggerService {
     /**
      * Write a 'debug' level log.
      */
-    debug?(message: any, ...optionalParams: any[]): any;
+    debug(message: any, ...optionalParams: any[]): any;
     /**
      * Write a 'verbose' level log.
      */
-    verbose?(message: any, ...optionalParams: any[]): any;
+    verbose(message: any, ...optionalParams: any[]): any;
     /**
      * Set log levels.
      * @param levels log levels
